app/template_folder/template_folder_schema.py

```diff
@@ -11,9 +11,9 @@
     "required": ["name", "parent_id"]
 }
 
-post_rename_template_folder_schema = {
+post_update_template_folder_schema = {
     "$schema": "http://json-schema.org/draft-04/schema#",
-    "description": "POST schema for renaming template_folder",
+    "description": "POST schema for updating template_folder",
```

Comment:
if we're making this generic, `name` shouldn't be required

Reply:
I think so, since it's the only field at the moment and otherwise it's an empty request